javaparser-core/pom.xml

```diff
@@ -107,7 +107,31 @@
                     </dependency>
                 </dependencies>
             </plugin>
+            <!-- Generate an OSGi-enabled MANIFEST during the build -->
+            <plugin>
+                <groupId>biz.aQute.bnd</groupId>
+                <artifactId>bnd-maven-plugin</artifactId>
+                <version>3.2.0</version>
```

Comment:
OK, I'll add that. Do you prefer me to push a second commit with the fix or squash the existing one?